docs/table/performance.inc.rst

```diff
@@ -39,3 +39,23 @@ then use ``serialize_method='data_mask'``.
 It uses the non-masked version of data and it is faster::
 
     >>> tm.write('tm.ecsv', overwrite=True, serialize_method='data_mask') # doctest: +SKIP
+
+Reading a ``.fits`` in |Table| using
+:meth:`~astropy.table.Table.read` when ``memmap=False`` can be very slow
+because it reads full |Table| in the memory ::
+
+    >>> import numpy as np
+    >>> from astropy.table import Table
+    >>> tbl = Table({
```

Comment:
Could you please reformat to an un-black way? I know it's fashionable tool right now, but I don't find having the closing parens in a new line, and overall spreading stuff over more lines etc making anything more readable.